aredis/pubsub.py

```diff
@@ -288,10 +288,11 @@ def run_in_thread(self, daemon=False):
 
 
 class PubSubWorkerThread(threading.Thread):
-    def __init__(self, pubsub, daemon=False):
+    def __init__(self, pubsub, daemon=False, poll_timeout=1.0):
```

Comment:
https://github.com/NoneGG/aredis/blob/a2b3ea8b1a957c1e243e26e662a323e04ed53ad3/aredis/pubsub.py#L285
Could you also add `poll_timeout` when initialize `PubSubWorkerThread` in `PubSub.run_in_thread`

Reply:
done, updated my PR